### ISSUE TITLE
DateTime cannot represent an invalid date-time-string

### ISSUE THREAD

**Issue by marcoreni** [Urigo/graphql-scalars#395]
I assume this is related to #380 

After updating to 1.2.1 from 1.1.5 I'm receiving this error:
`DateTime cannot represent an invalid date-time-string 2020-06-01T10:02:00+0000.`

I see that https://github.com/Urigo/graphql-scalars/blob/master/src/scalars/iso-date/validator.ts#L120 validation fails because my TZ is missing `:` (my backend is using RFC822 format); Javascript `Date.parse()` seems to be fine about it:
```
Date.parse("2020-06-05T09:37:00+0000")
1591349820000
Date.parse("2020-06-05T09:37:00+00:00")
1591349820000
```

In the previous version, the serializer "coerced" the non-iso input into a valid output, but this apparently is not the case anymore.

What is the expected way of handling DateTimes? Do I have to convert the date in the RFC3339-ISO8601 format in my resolvers?

(PS: Release notes are missing for 1.1.8 onward)

**Comment by Nainterceptor** [comment]
Same issue here. I've locked to 1.1.5

**Comment by ardatan** [comment]
The old behavior is available in 1.2.6!
